mvp.feature

```diff
@@ -191,295 +290,453 @@ Feature: User with no Author identity in the Authority Register for Personas see
+    Then they see that there are no Contributors registered
+    And a grid is shown with the fields
+      | Name           |
+      | Institution    |
+      | Corresponding  |
+      | Move up / down |
+      | Edit           |
+      | Delete         |
+    And a button to add an Author is enabled
+
+  @418
+  Scenario: User opens the add Author dialog
+    Given that the user navigates to <Contributor> tab
+    When they click the to add an Author
+    Then the dialog to add an Author is opened
+    And the dialog contains fields for
+      | Author-search      |
+      | Institution search |
+    And a button to add an Author is enabled
+
+  @419
+  Scenario: User adds an Author to the Author list
+    Given that the user opens the add Author dialog
+    And enters a Name in the Author search box
+    And see search results for the Author-search containing the last publication
+    And see the Institution from the Author Register
+    When they click the button to Add an Author
+    Then the dialog is closed
+    And the Author is shown in the Author grid in the Constributor tab
+
+  # Tegn delete og fullfør beskrivelse
+  @
+  Scenario: User creates a new Author in the Author dialog
+    Given that the user opens the add Author dialog
+    And they click the link to create a new Author
+    And enters a Name in the Author name box
+    And enter Institution details
+    When they click the button to Add an Author
+    Then the dialog is closed
+    And the Author is shown in the Author grid in the Constributor tab
+
+  #Legg til Scenario for delete, edit, move, korresponding
+
+  @
+  Scenario: User registers Corresponding Author
+    Given that the user registers initial metadata for a Publication based on a Link
+    And they have clicked Next
+    When they review the information for Contributors
+    Then they see that the Contributors are grouped in sections Authors and Other Contributors
+    And they set the correctvalues authors have the expected value for Corresponding Author
+    And they see that the Other Contributors are listed alphabetically by Surname
+    And they see that the Other Contributors have the correct Role, Name and Institution
+
+  # Corresponding Author is not in fact present in the Datacite data, but probably will be soon
 
   # Important note about the "License": This assumes that the article is published in a journal that has a special Unit
   # agreement, which entails that Unit has negotiated a contract for Norwegian corresponding authors
   # In all other cases, a License-picker is provided
 
-  Scenario: User adds license and files to a Publication
-    Given the user processes Contributor information for a Journal Article Publication data for a publication based on a Link
+  @275
+  Scenario: User navigates to files and licenses
+    Given the user processes Contributor information for a Publication in Journal Publication data for a publication based on a Link
```

Comment:
Related to Sarah's comment, publication-in-a-journal Publication data is the intended meaning, right?